cli.js

```diff
@@ -2,14 +2,16 @@
 'use strict';
 const meow = require('meow');
 const updateNotifier = require('update-notifier');
+const version = require('./lib/version');
+const interactiveui = require('./lib/interactiveui');
 const np = require('./');
 
 const cli = meow(`
 	Usage
 	  $ np <version>
 
 	  Version can be:
-	    patch | minor | major | prepatch | preminor | premajor | prerelease | 1.2.3
+	    ${version.SEMVER_INCREMENTS.join(' | ')} | 1.2.3
 
```

Comment:
We should have a description here that running just `np` starts the interactive UI.